Detectors/TRD/workflow/include/TRDWorkflow/TRDEventDisplayFeedSpec.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <gsl/span>
+
+#include "TRDBase/Geometry.h"
+#include "TRDBase/TrackletTransformer.h"
+
+#include "Framework/Task.h"
+#include "Framework/DataProcessorSpec.h"
+
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "DataFormatsTRD/TrackTRD.h"
+#include "DataFormatsTRD/TriggerRecord.h"
+#include "DataFormatsTRD/Digit.h"
+
+#include "ITStracking/json.h"
```

Comment:
Not urgent, but I wonder if it is worth moving this huge header to `Common/Utils/...` or somewhere else.

Reply:
Should I move it there in this PR?

Reply:
I don't think so. I don't know where it should really go.